server/app/services/webhook_service.py

```
@@ -352,6 +352,15 @@ async def process_call_report(
                 logger.warning(f"🔄 CALL STATE TRANSITION | Call {call_id}: {old_status} → Failed (source: webhook, reason: unknown ended_reason '{call.ended_reason}')")
                 logger.debug(f"Set call status to 'Failed' for unknown ended_reason: {call.ended_reason}")
             
+            # Update call attempts in customer_db after call completion
+            try:
+                from app.services.customer_db_service import CustomerDBService
+                await CustomerDBService.increment_call_attempts(db, call.service_record.phone_number, call.organization_id)
+                logger.debug(f"Incremented call attempts for phone {call.service_record.phone_number}")
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)